Source/WebCore/platform/graphics/MediaPlayer.h

```diff
@@ -576,6 +576,7 @@ class WEBCORE_EXPORT MediaPlayer : public MediaPlayerEnums, public ThreadSafeRef
     unsigned videoDecodedByteCount() const;
 
     void setPrivateBrowsingMode(bool);
+    bool privateBrowsingMode() const { return m_privateBrowsingMode; }
```

Comment:
Minor nit: this bool should verb. Maybe `isInPrivateBrowsingMode()` or just `inPrivate...`.